packages/corvid-cli/src/utils/electron.js

```diff
@@ -152,14 +153,27 @@ async function openWindow(app, windowOptions = {}) {
       setTimeout(resolve, isDevTools ? 1000 : 0);
     }).then(async () => {
       const corvidConfig = await readCorvidConfig(".");
-
+      win.webContents
+        .executeJavaScript(
+          "(() => ({santaBase: window.santaBase, editorBase: window.editorBase}))()"
```

Comment:
please verify that this is correct when passing a specific version or localhost in a query parameter

Reply:
done

Reply:
`santaBase
"https://localhost/"`